hex-arch-adapters-spring6/src/main/kotlin/dev/eskt/arch/hex/adapter/spring/EventListenerExecutorService.kt

```diff
@@ -44,7 +44,7 @@ public class EventListenerExecutorService(
     private val jobs = mutableMapOf<String, Job>()
 
     init {
-        val allEventListeners: List<EventListener> = singleStreamTypeEventListeners + multiStreamTypeEventListeners
```

Comment:
is kotlin inferring the type correctly here? we need a list of something that has an id

Reply:
Yea it infers correctly